pkg/grapicmd/config.go

```diff
@@ -38,104 +40,39 @@ func NewConfig(
 	prebuilt bool,
 	in io.Reader,
 	out, err io.Writer,
-) Config {
+) *Config {
+	v := viper.New()
 	afs := afero.NewOsFs()
 	rootDir, insideApp := fs.LookupRoot(afs, currentDir)
-	return &config{
-		v:          viper.New(),
-		fs:         afs,
-		currentDir: currentDir,
-		rootDir:    rootDir,
-		insideApp:  insideApp,
-		appName:    appName,
-		version:    version,
-		revision:   revision,
-		buildDate:  buildDate,
-		prebuilt:   prebuilt,
-		in:         in,
-		out:        out,
-		err:        err,
+	return &Config{
+		Fs:         afs,
+		CurrentDir: currentDir,
+		RootDir:    rootDir,
+		InsideApp:  insideApp,
+		AppName:    appName,
+		Version:    version,
+		Revision:   revision,
+		BuildDate:  buildDate,
+		Prebuilt:   prebuilt,
+		InReader:   in,
+		OutWriter:  out,
+		ErrWriter:  err,
+		viper:      v,
 	}
 }
 
-type config struct {
-	cfgFile             string
-	v                   *viper.Viper
-	fs                  afero.Fs
-	currentDir, rootDir string
-	insideApp           bool
-	appName, version    string
-	revision, buildDate string
-	prebuilt            bool
-	in                  io.Reader
-	out, err            io.Writer
-	readConfigErr       error
-}
-
-func (c *config) Init(cfgFile string) {
-	c.cfgFile = cfgFile
-	c.v.SetConfigFile(c.cfgFile)
-	c.readConfigErr = c.v.ReadInConfig()
-}
-
-func (c *config) Fs() afero.Fs {
-	return c.fs
-}
-
-func (c *config) CurrentDir() string {
-	return c.currentDir
-}
-
-func (c *config) RootDir() string {
-	return c.rootDir
-}
-
-func (c *config) IsInsideApp() bool {
-	return c.insideApp
-}
-
-func (c *config) AppName() string {
-	return c.appName
-}
-
-func (c *config) Version() string {
-	return c.version
-}
-
-func (c *config) Revision() string {
-	return c.revision
-}
-
-func (c *config) BuildDate() string {
-	return c.buildDate
-}
-
-func (c *config) Prebuilt() bool {
-	return c.prebuilt
-}
-
-func (c *config) InReader() io.Reader {
-	return c.in
-}
-
-func (c *config) OutWriter() io.Writer {
-	return c.out
-}
-
-func (c *config) ErrWriter() io.Writer {
-	return c.err
-}
-
-func (c *config) Package() string {
-	return c.v.GetString("package")
-}
-
-func (c *config) ServerDir() string {
-	return c.v.GetString("grapi.server_dir")
-}
-
-func (c *config) ProtocConfig() *protoc.Config {
+func (c *Config) Init(cfgFile string) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Config.Init should have comment or be unexported